modules/beacon/mkosi.build

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -eux
+
+# Create destination dir inside the image root
+mkdir -p "${DESTDIR}/usr/local/bin"
+
+ls -l "${SRCDIR}"
+
+BEACON_DIR="${SRCDIR}/beacon"
+
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+source "$HOME/.cargo/env"
+
+rustup target add x86_64-unknown-linux-musl
+
+cargo build  --release --manifest-path "${BEACON_DIR}/Cargo.toml" --target=x86_64-unknown-linux-musl
+
+cp "${BEACON_DIR}/target/x86_64-unknown-linux-musl/release/beacon" "${DESTDIR}/usr/local/bin/"
```

Comment:
Lots of files in this commit are missing trailing newlines.

Reply:
Rather than doing `mkdir ...; cp ...` use the `install` command which can do both at once.

Reply:
Does this end up installing the rust toolchain into the image? If so, we should avoid that. But it isn't obvious to me.